sasmodels/kernelcl.py

```diff
@@ -503,27 +488,22 @@ def __init__(self, q_vectors, dtype=generate.F32):
             self.q = np.empty(width, dtype=dtype)
             self.q[:self.nq] = q_vectors[0]
         self.global_size = [self.q.shape[0]]
-        self._cache_key = unique_id()
+        #print("creating inputs of size", self.global_size)
```

Comment:
PEP-8 does not specifically cover the case of commented-out code. I do not consider it to be an inline comment, and explicitly do not put an extra space before it as a reminder that it is code instead of comment.  

I often leave commented out print statements in the code; since they are for debugging and since bugs tend to cluster, I find that it is quicker to debug in future if I can simply uncomment the statements that already proved useful for debugging in the past.